examples/seismic/viscoacoustic/viscoacoustic_example.py

```diff
@@ -40,31 +39,36 @@ def run(shape=(50, 50), spacing=(20.0, 20.0), tn=1000.0,
 
 def test_viscoacoustic():
     _, _, _, [rec] = run()
-    assert np.isclose(norm(rec), 18.7749, atol=1e-3, rtol=0)
+    assert np.isclose(norm(rec), 684.4527, atol=1, rtol=0)
```

Comment:
What's going on here? This looks stange?

Reply:
These values for norm and tolerance are expected because we are considering the second-order SLS equation as a reference. We will fix it to compute all first and second-order equations tests.